spec/bolt/node/winrm_spec.rb

```diff
@@ -11,11 +11,14 @@
 
   let(:host) { ENV['BOLT_WINRM_HOST'] || 'localhost' }
   let(:port) { ENV['BOLT_WINRM_PORT'] || 55985 }
+  let(:ssl_port) { ENV['BOLT_WINRM_SSL_PORT'] || 55986 }
```

Comment:
I guess this is an automatic mapping (i.e. I don't see the `Vagrantfile` updated to map 5986 to 55986)?